src/acceptance/administrator/components/com_users/UserNotesCest.php

```diff
@@ -0,0 +1,162 @@
+		// Select User
+		$I->click(UserNotesForm::$selectUserButton);
+		$I->switchToIFrame('Select User');
+		$I->searchForItem($this->username);
+		$I->click(['link' => $this->name]);
+		$I->switchToPreviousTab();
+		// Select category
+		$I->click(UserNotesForm::$selectCategory);
+		$I->switchToIFrame('Select or Change Category');
+		$I->searchForItem($this->category);
+		$I->click(['link' => $this->category]);
+		$I->switchToPreviousTab();
+		// Fill editor
+		$I->scrollTo(UserNotesForm::$editor);
+		$I->scrollTo(UserNotesForm::$toggleEditor);
+		$I->click(UserNotesForm::$toggleEditor);
+		$I->fillField(UserNotesForm::$editor, $this->editorText);
+		// Save and close
+		$I->click(UserNotesForm::$dropDownToggle);
+		$I->clickToolbarButton('save & close');
+		// Assertion
+		$I->searchForItem($this->subject);
+		$I->see($this->subject);
+	}
+
+	/**
+	 * Unpublish User Notes
+	 *
+	 * @param   \AcceptanceTester  $I The AcceptanceTester Object
+	 *
+	 * @since   4.0.0
+	 *
+	 * @return  void
+	 */
+	public function unpublishUserNote(\AcceptanceTester $I)
+	{
+        	$I->wantToTest('that user notes are unpublished.');
+		$I->doAdministratorLogin();
+		$I->amOnPage(UserNotesList::$url);
+		$I->searchForItem($this->subject);
+		$I->click(UserNotesList::$option1);
+		$I->clickToolbarButton('unpublish');
+	}
+
+	/**
+	 * Publish user notes
+	 *
+	 * @param   \AcceptanceTester  $I The AcceptanceTester Object
+	 *
+	 * @since   4.0.0
+	 *
+	 * @return  void
+	 */
+	public function publishUserNote(\AcceptanceTester $I)
+	{
+        	$I->wantToTest('that user note is published');
+		$I->doAdministratorLogin();
+		$I->amOnPage(UserNotesList::$url);
+		$I->searchForItem($this->subject);
+		$I->click(UserNotesList::$option1);
+		$I->clickToolbarButton('publish');
+		// Assertion
+		$I->searchForItem($this->subject);
+		$I->see($this->subject);
+	}
+
+	/**
+	 * Check in user notes
+	 *
+	 * @param   \AcceptanceTester  $I The AcceptanceTester Object
+	 *
+	 * @since   4.0.0
+	 *
+	 * @return  void
+	 */
+	public function checkinUserNote(\AcceptanceTester $I)
+	{
+        	$I->wantToTest('that user note is checked in');
+		$I->doAdministratorLogin();
+		$I->amOnPage(UserNotesList::$url);
+		$I->searchForItem($this->subject);
+		$I->click(UserNotesList::$option1);
+		$I->clickToolbarButton('check-in');
+		// Assertion
+		$I->searchForItem($this->subject);
+		$I->see($this->subject);
+	}
+
+	/**
+	 * Trash user notes
+	 *
+	 * @param   \AcceptanceTester  $I The AcceptanceTester Object
+	 *
+	 * @since   4.0.0
+	 *
+	 * @return  void
+	 */
+	public function trashUserNote(\AcceptanceTester $I)
+	{
+        	$I->wantToTest('that user note is trashed');
+		$I->doAdministratorLogin();
+		$I->amOnPage(UserNotesList::$url);
+		$I->searchForItem($this->subject);
+		$I->click(UserNotesList::$option1);
+		$I->clickToolbarButton('trash');
+		// Assertion
+		$I->searchForItem($this->subject);
+		$I->dontSee($this->subject);
```

Comment:
This does not check that the note is trashed.

Reply:
It does, it removes the item from backend admin